transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -158,12 +159,12 @@ public EpollEventLoopGroup(int nThreads, Executor executor, EventExecutorChooser
     }
 
     /**
-     * @deprecated This method will be removed in future releases, and is not guaranteed to have any impacts.
+     * Sets the percentage of the desired amount of time spent for I/O in the child event loops.  The default value is
+     * {@code 50}, which means the event loop will try to spend the same amount of time for I/O as for non-I/O tasks.
      */
-    @Deprecated
     public void setIoRatio(int ioRatio) {
-        if (ioRatio <= 0 || ioRatio > 100) {
-            throw new IllegalArgumentException("ioRatio: " + ioRatio + " (expected: 0 < ioRatio <= 100)");
+        for (EventExecutor e: this) {
```

Comment:
This is done at the `EpollEventLoop` level and the approach is consistent with with `Kqueue` and `Nio`. We could move the check for `Epoll` and `Kqueue` up to the `EventLoopGroup` level because the `EventLoop` types are package private and this is the only current usage of the package private API, but I think it is reasonable to keep it consistent between the different approaches as this isn't expected to be called frequently.